app/controllers/room_guests_controller.rb

```diff
@@ -0,0 +1,12 @@
+class RoomGuestsController < ApplicationController
+  def create
+    room_guest = RoomGuest.create(room_guest_params)
+
+    redirect_to "/guests/#{room_guest_params[:guest_id]}"
```

Comment:
Nice job creating this 3rd controller - it's exactly the approach I like to take too.